SayTheirNames/Source/Controller/Home/HomeView.swift

```diff
@@ -42,15 +45,16 @@ final class HomeView: UIView {
         return customNavigationBar
     }()
     
-    lazy private(set) var locationCollectionView: UICollectionView = {
-        let locationLayout = UICollectionViewFlowLayout()
-        locationLayout.scrollDirection = .horizontal
-        locationLayout.sectionInsetReference = .fromContentInset
-        let inset = Theme.Components.edgeMargin
-        locationLayout.sectionInset = UIEdgeInsets(0, inset, 0, inset)
-        let locationCollectionView = UICollectionView(frame: .zero, collectionViewLayout: locationLayout)
-        locationCollectionView.contentInsetAdjustmentBehavior = .always
-        return locationCollectionView
+    lazy private(set) var locationCollectionView: UICollectionView? = {
+        return nil
+//        let locationLayout = UICollectionViewFlowLayout()
```

Comment:
Just a thought that we should keep `TODO` next to commented out code so it's easier to know that it's not dead code and when it should be implemented
```suggestion
//        TODO: Add back in when filters are enabled
//        let locationLayout = UICollectionViewFlowLayout()
```

Reply:
Left a comment on top of the file, might not be very visible. I'll commit this suggestion though anyway 👍